### PR TITLE
feat: issue v2 and v3 document

### PR DESCRIPTION
**Pull request by ChenRen93**
## Summary
Enable OpenAttestation version 2 and 3 document to be issued In 1 process flow

## Changes

- Added a configuration file that contains both version 2 and version 3 document 
- Modify publish job process flow for verifiable document batch and wrapping

## Issues

-

### CODE REVIEW

File: src/common/hook/useQueue/utils/publish.tsx [hunk 1]
```diff
@@ -97,7 +97,33 @@ export const getRawDocuments = async (forms: FormEntry[], config: Config): Promi
...
+};
+
+const processVerifiableDocument = async (
+  jobs: PublishingJob[],
+  nonce: number,
+  contractAddress: string,
+  verifiableDocuments: RawDocument[]
+) => {
+  if (verifiableDocuments.length === 0) return;
+  console.log(verifiableDocuments);
```
Comment (cavacado) [inline src/common/hook/useQueue/utils/publish.tsx:110]: remove

File: src/common/hook/useQueue/utils/publish.tsx [hunk 2]
```diff
@@ -97,7 +97,33 @@ export const getRawDocuments = async (forms: FormEntry[], config: Config): Promi
...
+  const rawDocuments = verifiableDocuments.map((doc) => doc.rawDocument);
```
Comment (cavacado) [inline src/common/hook/useQueue/utils/publish.tsx:111]: why do u do this again?

File: src/common/hook/useQueue/utils/publish.tsx [hunk 3]
```diff
@@ -97,7 +97,33 @@ export const getRawDocuments = async (forms: FormEntry[], config: Config): Promi
...
+    type: firstRawDocument.type,
+    nonce,
+    contractAddress,
+    documents: verifiableDocuments.map((doc, index) => ({
+      ...doc,
+      wrappedDocument: wrappedDocuments[index],
+    })),
+    merkleRoot: merkleRoot,
+    payload: {},
+  });
```
Comment (cavacado) [inline src/common/hook/useQueue/utils/publish.tsx:126]: dont mutate in a separate function, returns the jobs array and build the collection again later

File: src/common/hook/useQueue/utils/publish.tsx (hunk 2 above)
Comment (ChenRen93) [reply src/common/hook/useQueue/utils/publish.tsx:111]: inside rawDocument, there is other rawDocumentProperties. That give the correct OpenAttestation Raw Document

Comment (ChenRen93) [reply src/common/hook/useQueue/utils/publish.tsx:111]: let me rename to rawOpenAttestationDocument, will it be a bit clearer ? it is weird way of how the thing is structured

File: src/common/hook/useQueue/utils/publish.tsx (hunk 1 above)
Comment (ChenRen93) [reply src/common/hook/useQueue/utils/publish.tsx:110]: done

File: src/common/hook/useQueue/utils/publish.tsx (hunk 3 above)
Comment (ChenRen93) [reply src/common/hook/useQueue/utils/publish.tsx:126]: done

**Review by cavacado — APPROVED** [review]

**Pull request merged**